### DISCUSSION TITLE
Object Aggregates and Mereological (In)constancy

### DISCUSSION THREAD

**Discussion by gregfowlerphd** [JisooSeo/101-WG-Discussions#13] [General]
In the WG a few weeks ago, a member--sorry, I can't remember who--suggested that object aggregates in BFO are mereologically constant. Carter expressed skepticism, noting that BFO practitioners would classify schools of fish (for example) as object aggregates even though they're typically thought to change their membership across time.

While Carter's point is well-taken, it isn't conclusive. It might be the case that (i) object aggregate is the most natural BFO category into which to classify schools of fish, but (ii) in so classifying them, BFO is committed to a non-standard theory about their diachronic identity, according to which they cannot survive changes in their membership.

Thus, an explicit statement that BFO object aggregates needn't be mereologically constant would be helpful, and I recently found one. It appears in a paper by Barry entitled "Ontology of language, with applications to demographic data", which is available here: https://philpapers.org/archive/DOWOOL.pdf. Page 11 of the linked PDF states that "an [object] aggregate in BFO is able to preserve its identity while changing its members".

Just thought folks might be interested because it came up recently. :) Also, the paper's quite good, so it's nice to have an opportunity to popularize it.

**Comment by JisooSeo** [comment]
This is a great find! Thank you for the interesting point and the paper 😄

**Comment by gregfowlerphd** [comment]
Here's another example, from a paper by Barry entitled "On Classifying Material Entities in Basic Formal Ontology" (available here: https://philpapers.org/archive/SMIOCM.pdf):

> As is true for all material entities (for example: you), object aggregates may gain and lose parts while remaining numerically identical (one and the same individual) over time, and for some aggregates, especially in cases where membership is determined by fiat (for example a baseball team, a congressional committee) membership may change with time.

**Reply by JisooSeo** [reply]
I guess I have a hard time understanding how something is not mereologically constant while remaining numerically identical over time. I'll go ahead and read the paper! Thank you for the suggestion!

**Reply by gregfowlerphd** [reply]
Well, the standard view is that organisms like you, I, and my dog Lily (awarded to my ex during the divorce) all remain numerically identical over time despite changing our parts constantly. But maybe you just have Chisholmian intuitions. ;)

**Reply by JisooSeo** [reply]
Yeah, I just have a very strict concept of numerical identity but maybe I shouldn’t hold that position.

**Comment by gregfowlerphd** [comment]
I suppose it might be worth interrogating the claim that in BFO, object aggregates needn't be mereologically constant. If BFO is correctly thought of as a highly general model of reality consisting of, say, the set of definitions and axioms (and perhaps also elucidations, etc.) stated in the OWL file available at the BFO GitHub, then although this set is consistent with the claim that some object aggregates are mereologically inconstant, it is also consistent with the claim that all object aggregates are, of necessity, mereologically constant.
